### PR TITLE
Update release notes for odo

### PR DESCRIPTION
**Pull request by cdrage**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

/kind documentation
[skip ci]

**What does does this PR do / why we need it**:

Updates the release notes

**Which issue(s) this PR fixes**:

N/A

**PR acceptance criteria**:

N/A

**How to test changes / Special notes to the reviewer**:

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: docs/dev/development.adoc [hunk 1]
```diff
@@ -181,6 +181,9 @@ $ ./scripts/changelog-script.sh ${PREVIOUS_VERSION} ${NEW_VERSION}
 .. Create a PR to update `:tag` and `:revision` in the https://github.com/kadel/homebrew-odo/blob/master/Formula/odo.rb[`odo.rb`] file
 in https://github.com/kadel/homebrew-odo[`kadel/homebrew-odo`].
 . Create a PR and update the file `build/VERSION` with the  latest version number.
+. Create a blog post! Follow the https://github.com/openshift/odo/tree/master/docs/blog[template.md] file and push it to the website (a tutorial is located on the https://github.com/openshift/odo/tree/gh-pages[gh-pages] branch of odo
+. After the blog post, ideally the CHANGELOG in the release should be the same as the blog post. This is an example of a good release changelog: https://github.com/openshift/odo/releases/tag/v2.0.0
+. Add the the built site (including the blog post) to the release with `site.tar.gz` https://github.com/openshift/odo/tree/gh-pages#bundling-the-site-for-releases[using the publishing the site for releases guide].
```
Comment (mik-dass) [inline docs/dev/development.adoc:186]: `using the publishing site`

Comment (cdrage) [reply docs/dev/development.adoc:186]: Yes! Thank you.

File: docs/dev/development.adoc [hunk 2]
```diff
@@ -181,6 +181,9 @@ $ ./scripts/changelog-script.sh ${PREVIOUS_VERSION} ${NEW_VERSION}
 .. Create a PR to update `:tag` and `:revision` in the https://github.com/kadel/homebrew-odo/blob/master/Formula/odo.rb[`odo.rb`] file
 in https://github.com/kadel/homebrew-odo[`kadel/homebrew-odo`].
 . Create a PR and update the file `build/VERSION` with the  latest version number.
+. Create a blog post! Follow the https://github.com/openshift/odo/tree/master/docs/blog[template.md] file and push it to the website (a tutorial is located on the https://github.com/openshift/odo/tree/gh-pages[gh-pages] branch of odo
+. After the blog post, ideally the CHANGELOG in the release should be the same as the blog post. This is an example of a good release changelog: https://github.com/openshift/odo/releases/tag/v2.0.0
+. Add the the built site (including the blog post) to the release with `site.tar.gz` https://github.com/openshift/odo/tree/gh-pages#bundling-the-site-for-releases[using the bundling the site for releases guide].
```
Comment (girishramnani) [inline docs/dev/development.adoc:186]: the the twice

**Comment by girishramnani** [comment]
opened a new PR with changes needed https://github.com/openshift/odo/pull/4178 hence closing this

**Pull request closed without merging**